src/middleware/softDeleteMiddleware.js

```diff
@@ -19,5 +19,8 @@ export default {
     User: async (resolve, root, args, context, info) => {
       return resolve(root, setDefaults(args), context, info)
     }
+  },
+  Mutation: async (resolve, root, args, context, info) => {
+    return resolve(root, setDefaults(args), context, info)
```

Comment:
You set the defaults on ALL mutations? Seems wrong doesn’t it? @roschaefer

Reply:
No, that's intended. It doesn't hurt to set `deleted: false` and `disabled: false` unless it's provided.